app/services/submit_to_app_store/prior_authority/solicitor_payload_builder.rb

```diff
@@ -7,9 +7,9 @@ def initialize(application)
 
       def payload
         @application.solicitor.as_json(only: %i[
-                                         full_name
```

Comment:
are we intentionally dropping this field?

Reply:
Yes, it's intentional - we don't use the solicitor name field in PA, we use the solicitor contact name.